pydatalab/pydatalab/routes/v0_1/items.py

```diff
@@ -447,7 +447,9 @@ def delete_sample():
 delete_sample.methods = ("POST",)  # type: ignore
 
 
-def get_item_data(item_id, load_blocks=True):
+def get_item_data(item_id, load_blocks=False):
+    # load_blocks = request.args.get("load_blocks") or load_blocks
```

Comment:
```suggestion
def get_item_data(item_id, load_blocks: bool = False):
    """Generates a JSON response for the item with the given `item_id`,
    additionally resolving relationships to files and other items.
    
    Parameters:
       load_blocks: Whether to regenerate any data blocks associated with this
           sample (i.e., create the Python object corresponding to the block and 
           call its render function).
           
    """
```

Mostly for my benefit, does that description make sense?